### DISCUSSION TITLE
Serial pin labels?

### DISCUSSION THREAD

**Discussion by pghpete** [HefnySco/OBAL#2] [Q&A] [answered]
First, thank you for your work on this!  I am currently building a few of these boards (after getting them from JLCPCB) and eventually I would like to attach a GPS unit to the serial pins on the board.  Is there a pinout as to which pin is what for the 4 serial pins? I am sure I can dig it out from the gerber file but I was curious if it is listed somewhere else.

**Comment by HefnySco** [comment, marked as answer]
Hi, 

the four pins named serial in the bottom if the following image.

![image](https://user-images.githubusercontent.com/4184373/139753957-b87802e5-91c7-45f6-ae59-c4960b25981e.png)

connections are [RX,TX,GND,5V] from left to right. Remember  to connect RX<---> TX of GPS and vice versa.

![image](https://user-images.githubusercontent.com/4184373/139754076-1c205bff-344b-4c6b-8e65-12c8d6548b0e.png)

You need to run ardupilot file with -B /dev/serial0

**Reply by pghpete** [reply]
Fantastic! Thank you for the assistance and in an amazingly quick reply! I appreciate it greatly.
